docs/source/tutorials/playing-command.rst

```diff
@@ -2,37 +2,39 @@
 Playing with your command
 =========================
 
-Now that you have started :doc:`first-plugin`, maybe you want to write a more
-interesting command than the basic ``.hello`` one. Not that there is anything
-wrong with that command! The emoticons plugin is composed of commands like this
-one::
+Now that you have started :doc:`your first plugin <first-plugin>`, maybe you
+want to write a more interesting command than the basic ``.hello`` one. Not
+that there is anything wrong with that command! The emoticons plugin is
+composed of commands like this one::
 
     @plugin.command('shrug')
     @plugin.action_command('shrugs')
     def shrug(bot, trigger):
         bot.say('¯\\_(ツ)_/¯')
 
-Which is one of the maintainers' favorite command to use. However, let's see
-if we can do a bit more complex than that.
+Which is one of the maintainers' favorite commands to use. However, let's see
+if we can do something a bit more *complex* than that.
+
 
 Greeting a user by their name
 =============================
 
-Have you noticed that a :func:`plugin_callable` takes two arguments? The first
-one is the ``bot``, i.e. an instance of Sopel (wrapped for your convenience)
-that you can use to :doc:`/plugin/bot`.
+Have you noticed that a :ref:`plugin callable <Plugin callables>` takes **two
+arguments?** The first one is the ``bot``, i.e. an instance of Sopel (wrapped
+for your convenience) that you can use to
+:doc:`interact with the bot </plugin/bot>`.
 
 In the previous tutorial, we used ``bot.reply``, which is convenient but not
 always what you want. Maybe you want the bot to say this::
 
     <YourNick> .hello
     <Sopel> Hello YourNick, have a nice day!
 
-For that, you need the second argument: the ``trigger``. It is an object with
-all the information you need about the message that triggered the callable,
-such as the message itself, the channel, the type of message, etc. and what we
-need for now is the :attr:`trigger.nick <sopel.trigger.Trigger.nick>`
-attribute::
+For that, you need **the second argument**: the ``trigger``. It is an object
+with all the information you need about the message that triggered the
```

Comment:
```suggestion
with all the information you need about the message that triggered your
```

Not that the previous wording was _wrong_, but in the context of a tutorial I think "your" makes more sense than "the".

Reply:
Adding on to @dgw's suggestion, I think it makes sense to state more directly that the `trigger` holds information about the message, and the rest of the example speaks for the words this edit omits.

```suggestion
with information about the message that triggered your
```

Reply:
This is tough to decide, but ultimately I prefer a more direct approach too.